--- functions/fisher.fish	2021-03-23 19:07:12.982569554 +0700
+++ tmp/fisher.fish	2021-03-23 18:57:22.977513060 +0700
@@ -84,6 +84,12 @@
                         set name (string split \@ $plugin) || set name[2] HEAD
                         set url https://codeload.github.com/\$name[1]/tar.gz/\$name[2]
 
+                        if string match -q 'gitlab.com/*' \$name[1]
+                            set bare (string replace 'gitlab.com/' '' \$name[1])
+                            set repo (string split '/' \$bare)
+                            set url https://gitlab.com/{\$bare}/-/archive/{\$name[2]}/{\$repo[-1]}-{\$name[2]}.tar.gz
+                        end
+
                         echo Fetching (set_color --underline)\$url(set_color normal)
 
                         if curl --silent \$url | tar -xzC \$temp -f - 2>/dev/null
